package.json

```diff
@@ -1,23 +1,22 @@
 {
-  "name": "monowingo",
+  "name": "@winglang/monorepo",
+  "private": true,
   "devDependencies": {
-    "@nrwl/devkit": "16.5.0",
     "bump-pack": "workspace:^",
     "changelogen": "^0.5.3",
-    "nx": "16.5.0",
-    "semver": "^7.3.8"
+    "semver": "^7.3.8",
+    "turbo": "^1.10.7"
```

Comment:
Would be nice to update this to at least 1.10.10 before merging, looks like there is a useful fix for `workspace:` references: https://github.com/vercel/turbo/releases/tag/v1.10.10